pallets/parachain-staking/src/lib.rs

```diff
@@ -121,6 +121,9 @@ pub mod pallet {
 		/// Minimum number of blocks per round
 		#[pallet::constant]
 		type MinBlocksPerRound: Get<u32>;
+		///If a collator doesn't produce any block on this number of rounds, it is marked as offline
+		#[pallet::constant]
+		type NumberRoundsOffline: Get<u32>;
```

Comment:
```suggestion
		type MaxOfflineRounds: Get<u32>;
```

I suggest a more explicit name